### ISSUE TITLE
Would you mind providing data of the corresponding logits for models you trained?

### ISSUE THREAD

**Issue by GuangyuanHao** [AmirooR/IntraOrderPreservingCalibration#2]

**Comment by GuangyuanHao** [comment]
I mean the logits before conlibration. Thanks a lot.

**Comment by AmirooR** [comment]
@GuangyuanHao Please take a look here: https://github.com/AmirooR/IntraOrderPreservingCalibration/issues/1#issuecomment-757445248

Note that (logits_val, y_val) in https://github.com/AmirooR/IntraOrderPreservingCalibration/blob/main/builder.py#L56 are logits and labels before calibration. I will add the remaining logits soon.

**Comment by GuangyuanHao** [comment]
Thanks for your reply. But how can I download the corresponding pickle files?

**Comment by AmirooR** [comment]
I've just uploaded them to google drive. You can download all of the pickle files from this link: https://drive.google.com/file/d/18QvhqtfUY77xV7tZa25q4h4BLsbeFmxV/view?usp=sharing

Some of them are taken from [this repo](https://github.com/markus93/NN_calibration) as mentioned in https://github.com/AmirooR/IntraOrderPreservingCalibration/issues/1#issuecomment-757445248.

**Comment by GuangyuanHao** [comment]
Thanks for your help very much!
